extensions/wikia/Insights/scripts/InsightsIndexPageTracking.js

```diff
@@ -0,0 +1,26 @@
+require(['wikia.tracker'],
+	function (tracker) {
+		'use strict';
+
+		function onLinkClick() {
```

Comment:
What do you think about using ```buildTrackingFunction``` from wikia tracker and prepare params outside ```onLinkClick``` function and pass only label?